contracts/test/integration/vip/vip4.sol

```diff
@@ -0,0 +1,186 @@
+pragma solidity =0.8.13;
+
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {TimelockSimulation} from "../utils/TimelockSimulation.sol";
+import {MainnetAddresses} from "../fixtures/MainnetAddresses.sol";
+import {ArbitrumAddresses} from "../fixtures/ArbitrumAddresses.sol";
+import {DSTest} from "./../../unit/utils/DSTest.sol";
+import {Core} from "../../../core/Core.sol";
+import {Vm} from "./../../unit/utils/Vm.sol";
+import {IVIP} from "./IVIP.sol";
+import {AllRoles} from "./../utils/AllRoles.sol";
+
+contract vip4 is DSTest, IVIP, AllRoles {
```

Comment:
Nite: Would capitalise the first letter

Reply:
Capitalizing things causes either forge or hardhat compiler to brick, so I just lowercased everything to bypass this.